k6/src/scenarios/test/bootstrapEnvParameters.js

```diff
@@ -25,6 +25,8 @@ import {
   setDefaultValuesForEnvParameters
 } from "../../lib/parameters.js";
 
+const scParams = JSON.parse(open("../../prepare/.smartContractParams.json"));
```

Comment:
Can you explain where this file comes from?  Is it generated?

Reply:
yes, it is generated by the npm `prep` task, that is basically a bootstrap for the testing.